### ISSUE TITLE
About datasets of your experiments.

### ISSUE THREAD

**Issue by daikitemp** [penghu-cs/MRL#5]
Dear Peng Hu,

Sorry to bother you again. 
There's something I'd like to ask about datasets of your experiments.

In your paper, in addition to "Wikipedia", there were experiments on "INRIA-Websearch", "NUS-WIDE", and "XmediaNet"." The data from "wikipedia" is available on your github, but the data from other datasets are not available. Is it possible to make those datasets available? (i.e., nus_wide_deep_doc2vec_data_42941.h5py, INRIA-Websearch.mat in noisydataset.py, etc.)
If this is difficult, could you please tell me how to create them?

I am sorry for taking up so much of your time but I appreciate your help.

Thanks so much.

**Comment by penghu-cs** [comment]
Dear Daiki Okamura,

Because we should apply authorization for these datasets from the authors,
I do not think it would be proper to share the datasets online. These
datasets could access through the following websites. You could download
the raw data and train with the corresponding backbones (image backbone are
the pre-trained models given by the torchvision, and the pre-trained
doc2vec model is available from https://github.com/jhlau/doc2vec). Attached
is the list for  nus_wide_deep_doc2vec_data_42941. Thanks a lot for your
attention. Hope this helps.

Best regards,
Hu, Peng

INRIA-Websearch: https://weiyc.github.io/assets/projects/tcyb.html
NUS-WIDE:
https://lms.comp.nus.edu.sg/wp-content/uploads/2019/research/nuswide/NUS-WIDE.html
XmediaNet: http://59.108.48.34/tiki/XMediaNet/

NUS-WIDE

Daiki Okamura ***@***.***> 于2021年11月12日周五 下午8:44写道：

**Comment by daikitemp** [comment]
Dear Peng Hu,

Thank you for your prompt reply.
I understand that it is not possible to put it on the site due to authorization.
Thank you also for sharing the URL of the Dataset.
There are two additional questions I would like to ask you.
First, which model did you use for the pre-trained Doc2Vec, "English Wikipedia" or "Associated Press News"?
Secondly, I'm not familiar with the NUS-WIDE dataset, but am I correct in assuming that I need to download the Flickr dataset separately?

I am sorry to ask you so many times but I appreciate your help.

Thanks so much.

**Comment by penghu-cs** [comment]
Dear Daiki Okamura,

Thank you for your understanding.
1. We used the pre-trained Doc2Vec, which has been trained on "English
Wikipedia".
2. Yes. You could download the Flickr dataset, and construct NUS-WIDE with
the given lists.

Thanks a lot.

Best regards,
Hu, Peng

Daiki Okamura ***@***.***> 于2021年11月15日周一 上午9:22写道：

**Comment by daikitemp** [comment]
Dear Peng Hu,

Thank you very much for your answer.
Your answer is very understandable.
I appreciate your kind answer.

Thanks you so much.

**Comment by penghu-cs** [comment]
Dear Daiki Okamura,

You are welcome. Thanks very much for your attention.

Best regards,
Hu, Peng

Daiki Okamura ***@***.***> 于2021年11月15日周一 下午12:07写道：

**Comment by daikitemp** [comment]
Dear Peng Hu,

Thank you very much for your detailed answer the other day.
I would like to ask you two additional questions about your dataset.

First, you mentioned that you used pre-trained Doc2Vec for text feature extraction. How many epochs did you set for feature extraction (i.e., epochs or steps in gensim.models.Doc2Vec.infer_vector())?
Secondly, INRIA-Webserch has 'fc6', 'fc7', 'FT_fc6', 'FT_fc7' as image features, which feature did you use in your experiment?

I’m really sorry to bother you over and over but I appreciate your help.

Sincerely,
Daiki Okamura

**Comment by penghu-cs** [comment]
Dear Daiki Okamura,

Thanks a lot for your attention. The answers are attached as follows:
1. The hyperparameters are set as:   alpha= 0.01 and steps= 1000  , i.e.,
Doc2Vec.infer_vector(s, alpha= 0.01, steps= 1000).
2. 'fc6' is used as image features for INRIA-Webserch.
Thanks.

Best regards,
Hu, Peng

Daiki Okamura ***@***.***> 于2021年11月29日周一 下午2:01写道：

**Comment by daikitemp** [comment]
Dear Peng Hu,

Thank you for your quick response.
Your answers are very helpful and easy to understand.
It helped me a lot.

Thanks you so much.

Sincerely,
Daiki Okamura

**Comment by daikitemp** [comment]
Dear Peng Hu,

I am sorry to keep repeating myself but I'm very interested in your method.
I would like to ask you about hyperparameters and datasets.

First of all, about the hyperparameter, in equation (8) of your paper, there is a hyperparameter β. Can you tell me the optimal value of β for your experiments on each dataset? Or is β a constant value for all experiments?
One more thing about the dataset, did you use 'tr_text' or 'te_text' as text features directly in INRIA-Websearch?

I am sorry to ask you so many times but I appreciate your help.

Sincerely,
Daiki Okamura

**Comment by penghu-cs** [comment]
Dear Daiki Okamura,

You are so polite! Thanks a lot for your interest. This method is a bit
sensitive to the hyperparameter β. We tuned the parameter on the validation
set of different datasets.
Yes.  The 'tr_text' or 'te_text' of the given data are used as text
features in our experiments.

If you have any questions, please let me know. Thanks.

Best wishes,
Hu, Peng

Daiki Okamura ***@***.***> 于2021年12月9日周四 19:12写道：

**Comment by daikitemp** [comment]
Dear Peng Hu,

Thank you very much for your kindness every time.

I understand that hyperparameters are tuned with a validation set.
Also, thank you for your answer about datasets.
As an additional question, did you do any preprocessing on the text when doing feature extraction on Xmedia or Wikipedia dataset?

Thank you very much for your kind answer.

Sincerely,
Daiki Okamura

**Comment by daikitemp** [comment]
Dear Peng Hu,

It’s been a while since our last correspondence.
I have two questions about the NUS-WIDE dataset.

First, you said that you selected 10 classes from 81 classes (each sample belongs to only one of ten classes) in the NUS-WIDE dataset used in your experiment.
Could you tell me which classes you selected?

Secondly, I understand the NUS-WIDE dataset is a multi-label dataset.
Is there any overlap of classes in the selected data?

We would appreciate your answer.

Sincerely,
Daiki Okamura

**Comment by penghu-cs** [comment]
Dear Daiki Okamura,

Sorry for the late reply. The data are selected following the list given by
the authors of [1]. Because of the license, it is not proper to share the
data online. Some details could be found in the paper. Thanks.

[1] Yuxin Peng, Jinwei Qi, Xin Huang, and Yuxin Yuan. 2018. CCL:
Cross-modal Correlation Learning With Multigrained Fusion by Hierarchical
Network. IEEE Transactions on Multimedia 20, 2 (Feb 2018), 405–420.

Best regards,
Hu, Peng

Daiki Okamura ***@***.***> 于2022年6月3日周五 17:18写道：

**Comment by daikitemp** [comment]
Dear Peng Hu,

Thank you for your reply.
I understand that you can't share because of the license.
Furthermore, thank you for sharing the reference.
I refer to it.

Thank you for your cooperation.

Sincerely,
Daiki Okamura
